### DISCUSSION TITLE
New Builds Q3 2024

### DISCUSSION THREAD

**Discussion by ianw1974** [ianw1974/zimbra-build-scripts#88] [Announcements]
Latest builds for Q3 2024 now available on my site as usual.

**Comment by jeredfloyd** [comment]
Thanks!  Is there any easy way to tell how these differ (in source commits) from the previous 10.0.8 builds?  

Zimbra pushed FOSS updates to the 10.1 repo a few weeks ago but haven't yet done a corresponding release.

**Reply by ianw1974** [reply]
Not really.  The build process clones 55 different repositories, all from develop branch.  It would require comparing every single one for changes between the last build date and the current one for each of those repositories.  I'm sure there will be changes that the previous builds do not have.  I doubt that the development has been stagnant on Zimbra's side for there to be no changes.
